src/main/java/gregtech/api/render/MetaTileEntityTESR.java

```diff
@@ -125,10 +125,21 @@ public void renderTileEntityFast(MetaTileEntityHolder te, double x, double y, do
 
     @Override
     public boolean isGlobalRenderer(MetaTileEntityHolder te) {
-        if(te instanceof IRenderMetaTileEntity) {
-            return ((IRenderMetaTileEntity) te).isGlobalRenderer();
-        } else if(te instanceof IFastRenderMetaTileEntity) {
-            return ((IFastRenderMetaTileEntity) te).isGlobalRenderer();
+        if (te instanceof IRenderMetaTileEntity) {
```

Comment:
I think you need to resync your working branch with master.
This change is already applied in #1563

Reply:
well, my negligence